<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userMatchSettingsWindow</class>
 <widget class="QDialog" name="userMatchSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNumOfGames">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumOfGamesTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Of Games In Match:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrNumOfGames">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGameTimeLimit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeLimitTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Time Limit (min):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrGameTimeLimit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxScoreThreshold">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelScoreThresholdTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Score Threshold:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrScoreThreshold">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGameVicType">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGameVicTypeTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Victory Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrGameVicType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>userMatchSettingsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>userMatchSettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
